### PR TITLE
Sascala toc edit

### PR DESCRIPTION
**Pull request by sascala**
I've made a first draft of substantial organization changes to the Marathon docs site. I'd love your feedback:

* Do you think there should be any other top-level categories?
* Are the topics I've put under "Deploying Applications" and "Management" correctly placed?
@aquamatthias, @gkleiman, @kolloch: I'd worked with @meichstedt on this, so I assigned the PR to him, but I'd very interested in all of your feedback.

The relevant file is in `docs/_layouts/docs.html`

### CODE REVIEW

File: docs/docs/index.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
----
-title: Setting Up and Running Marathon
----
+-----------------------
+title: Install Marathon
+-----------------------
```
Comment (meichstedt) [inline docs/docs/index.md]: Note: wrapping the title in two lines containing three dashes `---` each is a markup syntax that results in html metadata containing the key/value pairs in between them. If you use more than 3 dashes, the syntax is broken and does **not** result in the wanted metadata. This also breaks html generation at various points for me. Please fix this in all the changed *.md files

File: docs/_layouts/docs.html [hunk 2]
```diff
@@ -3,193 +3,227 @@
...
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/high-availability.html">
+                    High Availability Mode
+                </a>
+            </li>
+        </ul>
+
+        <h5>Application Deployment</h5>
```
Comment (meichstedt) [inline docs/_layouts/docs.html]: Problem: _Application Deployment_ is a rather abstract term. All the pages in this section are/should be about using Marathon, so I guess _Using Marathon_ would be a suitable header. Does that make sense to you?

File: docs/_layouts/docs.html [hunk 3]
```diff
@@ -3,193 +3,227 @@
...
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/recipes.html">
+                    Recipes
+                </a>
+            </li>
+        </ul>
+
+        <h5>Management</h5>
```
Comment (meichstedt) [inline docs/_layouts/docs.html]: To me, It's not really clear from the header whether this section is about managing Marathon or managing application deployments with marathon or something else. The current content is a mixture of _managing application deployments with Marathon_ (e.g. _Application Groups_, _Blue Green Deployments_, _Running Docker Containers_) and _Administrating Marathon_ (_SSL and Basic Auth_, _Framework Auth_, _Service Discovery_).

I'd put everything that covers managing applications and their deployments under _Using Marathon_ (see above), and the rest under _Configuring Marathon_ (or _Administrating Marathon_ if that's more specific).

Again: Does that make sense to you?

File: docs/_layouts/docs.html [hunk 4]
```diff
@@ -3,193 +3,227 @@
...
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/command-line-flags.html">
+                    Command Line Flags
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/developing-vm.html">
+                    Run a Local Version of Marathon
```
Comment (meichstedt) [inline docs/_layouts/docs.html]: I suggest to put this under _Development_ – it not about using Marathon and not about administration either, mostly helpful for someone that wants to run in locally for development or testing purposes.

**Comment by meichstedt** [comment]
Btw: your branch is 234 commits behind master. Please rebase! There have been changes to `metrics.md` in the meantime that conflict with your changes to that file; unfortunately you'll have to resolve those conflicts.

Seems like you used an old existing branch to add your changes. In general, it's easer to create a new branch based on master and commit to that one.

**Comment by meichstedt** [comment]
Something strange happened to metrics.md. Which editor are you using?

**Comment by meichstedt** [comment]
In general: I think structuring the pages like that is a step in the right direction. As described above, I'd like more specific terms like _Using Marathon_ vs. _Administrating Marathon_, and you need to clearly separate between these 2 sections. I'll help you organizing them.

**Comment by meichstedt** [comment]
There are 8 commits in this PR – which ones do you actually want to merge?

File: docs/docs/index.md (hunk 1 above)
Comment (sascala) [inline docs/docs/index.md]: Sorry about that. Will change.

File: docs/_layouts/docs.html (hunk 2 above)
Comment (sascala) [inline docs/_layouts/docs.html]: Yes.

File: docs/_layouts/docs.html (hunk 3 above)
Comment (sascala) [inline docs/_layouts/docs.html]: It seems like "Administration" would be more accurate than configuration. For a few of these topics I hesitated about whether they would be more appropriate to "using" or "administering". If you see things misplaced, let me know!

File: docs/_layouts/docs.html (hunk 4 above)
Comment (sascala) [inline docs/_layouts/docs.html]: Ah OK. So an application developer wouldn't need to do this, only someone developing Marathon itself.

**Comment by sascala** [comment]
@meichstedt Thanks for your comments. I'll update the file with your suggestions. AFAICT I rebased and negotiated the merge conflicts -- was I successful?

**Comment by aquamatthias** [comment]
@sascala there are a couple of commits, that do not belong to this PR. 
After rebasing can we merge?

**Comment by meichstedt** [comment]
Closing this in favor of #3746

**Pull request closed without merging**